src/test/ui/async-await/issue-69663-lingering-local-borrows.rs

```diff
@@ -0,0 +1,51 @@
+// edition:2018
+// run-pass
```

Comment:
`build-pass` is probably enough here, or probably even `check-pass` (which I think is the default if you don't do anything).

Reply:
I think `build-pass` is better. It seems that there might be changes to MIR outputs.